test/unit/migrations/036-test.js

```diff
@@ -0,0 +1,129 @@
+const assert = require('assert')
+let id = 0
+const migrationTemplate = require('../../../app/scripts/migrations/036')
+const nullStorage = {meta: { version: 0 }, data: {}}
+const storage = {
+  meta: {},
+  data: { TransactionController: {
+    transactions: [],
+  } },
+}
+
+const counts = {}
+
+let status = [
+  'unapproved',
+  'rejected',
+  'approved',
+  'signed',
+  'submitted',
+  'confirmed',
+  'failed',
+  'dropped',
+]
+const nets = [ 1, 2, 3, 4, 5]
+let currentStatus = status.pop()
+let nid = 0
```

Comment:
what does 'nid' stand for?

Reply:
sorry i was being lazy and didnt want to type out something like `netIdIndex` and brain farted to `nid` i can change it to `i` or `index` if you want

Reply:
`netIdIndex` plz, easy to confuse `nid` for `netId`